client/src/components/Tickets/TicketModal.js

```diff
@@ -170,6 +182,18 @@ const TicketHistoryCard = ({
       </CardHeader>
       <Collapse isOpen={isOpen}>
         <CardBody>{content}</CardBody>
+        <CardBody
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(attachments = (attachments || []).filter(
+            a => !a[1].toString().startsWith('untitled (')
+          ))}
+        >
+          Attachments:
+          {attachments &&
+            attachments.map(attachment => (
+              <Attachment attachment={attachment} />
```

Comment:
It seems this would benefit from a refactor to being an `<Attachments>` component instead of multiple `<Attachment>`s, that way we can keep all attachments in the same `<ul>`